tests/terratest/functions/set/setRKE1.go

```diff
@@ -0,0 +1,163 @@
+package functions
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/hashicorp/hcl/v2/hclsyntax"
+	"github.com/hashicorp/hcl/v2/hclwrite"
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/pkg/config"
+	format "github.com/rancher/rancher/tests/terratest/functions/format"
+	"github.com/rancher/rancher/tests/v2/validation/terratest"
+	"github.com/zclconf/go-cty/cty"
+)
+
+func SetRKE1(k8sVersion string, nodePools []terratest.Nodepool, file *os.File) (done bool, err error) {
+	rancherConfig := new(rancher.Config)
+	config.LoadConfig("rancher", rancherConfig)
+
+	terraformConfig := new(terratest.TerraformConfig)
+	config.LoadConfig("terraform", terraformConfig)
+
+	// create new empty hcl file object
```

Comment:
Same nit for comments.

Reply:
Is the team consensus to remove these comments?

I don't see the benefit in removing these comments. Anyone referencing the code will save time being able to scan the comments for the appropriate section, and it doesn't cause clutter IMO. As we grow in features, we will likely need to add new fields to these blocks for future feature support, so I see them being beneficial in the long run.